src/biguint.rs

```diff
@@ -1420,7 +1420,7 @@ macro_rules! impl_rem_assign_scalar {
             fn rem_assign(&mut self, other: &BigUint) {
                 *self = match other.$to_scalar() {
                     None => *self,
-                    Some(0) => panic!(),
+                    Some(0) => panic!("attempt to calculate the remainder with a divisor of zero"),
```

Comment:
Let's keep the various `Rem` impls consistent -- elsewhere it's using `div_rem`.

```suggestion
                    Some(0) => panic!("attempt to divide by zero"),
```